dali/dfuplus/dfuplus.cpp

```diff
@@ -895,6 +895,9 @@ int CDfuPlusHelper::copy()
     else if(globals->hasProp("noRecover"))
         req->setNorecover(globals->getPropBool("noRecover", false));
 
+    if(globals->hasProp("expireDays"))
+        req->setExpireDays(globals->getPropInt("expireDays"));
```

Comment:
looks right, but this is a slightly different issue right?
In that it was impossible to set expireDays via a dfuplus copy command line before this change?

Reply:
Changed